[2025-01-02T14:31:29.072+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-02T14:31:29.106+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gold_finance_data_pipeline.fetch_max_date_task manual__2025-01-02T14:31:20.237220+00:00 [queued]>
[2025-01-02T14:31:29.121+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gold_finance_data_pipeline.fetch_max_date_task manual__2025-01-02T14:31:20.237220+00:00 [queued]>
[2025-01-02T14:31:29.121+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 3
[2025-01-02T14:31:29.146+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_max_date_task> on 2025-01-02 14:31:20.237220+00:00
[2025-01-02T14:31:29.156+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=325) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-02T14:31:29.161+0000] {standard_task_runner.py:72} INFO - Started process 327 to run task
[2025-01-02T14:31:29.165+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'gold_finance_data_pipeline', 'fetch_max_date_task', 'manual__2025-01-02T14:31:20.237220+00:00', '--job-id', '325', '--raw', '--subdir', 'DAGS_FOLDER/Gold_finance_etl_dags.py', '--cfg-path', '/tmp/tmp21tzjwnn']
[2025-01-02T14:31:29.169+0000] {standard_task_runner.py:105} INFO - Job 325: Subtask fetch_max_date_task
[2025-01-02T14:31:29.290+0000] {task_command.py:467} INFO - Running <TaskInstance: gold_finance_data_pipeline.fetch_max_date_task manual__2025-01-02T14:31:20.237220+00:00 [running]> on host 77bcd09e75b5
[2025-01-02T14:31:29.436+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gold_finance_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_max_date_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-02T14:31:20.237220+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-02T14:31:20.237220+00:00'
[2025-01-02T14:31:29.438+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-02T14:31:29.457+0000] {logging_mixin.py:190} INFO - Error fetching max date: name 'PostgresHook' is not defined
[2025-01-02T14:31:29.457+0000] {python.py:240} INFO - Done. Returned value was: 2000-01-01 00:00:00
[2025-01-02T14:31:29.492+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-02T14:31:29.493+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=gold_finance_data_pipeline, task_id=fetch_max_date_task, run_id=manual__2025-01-02T14:31:20.237220+00:00, execution_date=20250102T143120, start_date=20250102T143129, end_date=20250102T143129
[2025-01-02T14:31:29.541+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-02T14:31:29.580+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-02T14:31:29.585+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
